protos/sigstore_trustroot.proto

```diff
@@ -231,7 +243,8 @@ message SigningConfig {
 // Clients MUST use the API version hint to determine the service with the
 // highest API version that the client is compatible with. Clients MUST also
 // only connect to services within the specified validity period and that has the
-// newest validity start date.
+// newest validity start date. Clients MUST select services from distinct operators
+// when selecting multiple services.
```

Comment:
I think we should provide a suggested algorithm for selection, it's starting to get pretty confusing.